### ISSUE TITLE
GPU acceleration does not work (GeminiLake / Celeron N4000 - octopus meep - Debian 12 Stable w/ Linux 4.14)

### ISSUE THREAD

**Issue by kkilobyte** [ading2210/shimboot#45]
**Describe the bug**
When trying to enable GPU acceleration using the apt and echo commands found in the FaQ of the readme, GPU acceleration will not enable with `i915` or `i965` driver.

**To Reproduce**
1. Run `sudo apt install libglx-amber0 libegl-amber0 && echo "MESA_LOADER_DRIVER_OVERRIDE=i965" | sudo tee -a /etc/environment` or replace `i965` with `i915`.
2. Reboot the system.
3. Try to do anything with animations or constant screen refreshing (like opening the app menu or typing quickly).
4. See screen tearing and jittery or no animations.

**Expected behavior**
Any animations should be smooth and visible while not seeing any screen tearing.

**Screenshots / Photos**
https://github.com/user-attachments/assets/24ee73a4-a027-41de-845d-084f95bbe8bb

**Target ChromeOS Device (please complete the following information):**
 - HP Chromebook x360 11 G2 EE (octopus - meep)
 - Shimboot v1.1.1
```
       _,met$$$$$gg.          kilo@meepuwu 
    ,g$$$$$$$$$$$$$$$P.       ------------ 
  ,g$$P"     """Y$$.".        OS: Debian GNU/Linux 12 (bookworm) on Chrome OS x86_64 
 ,$$P'              `$$$.     Host: Meep rev3 
',$$P       ,ggs.     `$$b:   Kernel: 4.14.91-18023-g2ab161c540baf 
`d$$'     ,$P"'   .    $$$    Uptime: 40 mins 
 $$P      d$'     ,    $$P    Packages: 1914 (dpkg) 
 $$:      $$.   -    ,d$$'    Shell: fish 3.6.0 
 $$;      Y$b._   _,d$P'      Resolution: 1366x768 
 Y$$.    `.`"Y$$$$P"'         DE: Plasma 5.27.5 
 `$$b      "-.__              WM: KWin 
  `Y$$                        Theme: [Plasma], catppuccin-mocha-mauve-standard+default [GTK2/3] 
   `Y$$.                      Icons: ePapirus-Dark [Plasma], ePapirus-Dark [GTK2/3] 
     `$$b.                    Terminal: konsole 
       `Y$$b.                 CPU: Intel Celeron N4000 (2) @ 2.600GHz 
          `"Y$b._             GPU: Intel GeminiLake [UHD Graphics 600] 
              `"""            Memory: 1520MiB / 3805MiB
```

**Additional context**
I am using a prebuilt v1.1.1 image from the releases tab and I have replaced LightDM and XFCE4 with SDDM and KDE 5. I have Debian itself on the eMMC while using a SD card as the bootloader, and used `dpkg-reconfigure` to change the timezone from UTC to PDT.

**Comment by ading2210** [comment]
Can you send your Xorg.0.log file?

**Comment by kkilobyte** [comment]
> Can you send your Xorg.0.log file? If you're using wayland currently try Xorg.

everytime i tried to use wayland i just got a black screen so i just assumed the shim kernel was too old for wayland

either way here is the [Xorg.0.log file](https://github.com/user-attachments/files/16337779/Xorg.0.log)

**Comment by ading2210** [comment]
Did GPU acceleration work *before* you switched out XFCE for KDE?

**Comment by kkilobyte** [comment]
i'll recheck but xfce did have screen tearing but i passed that off as xfce being xfce

**Comment by kkilobyte** [comment]
yeah no switching back to xfce made no difference even after switching back to lightdm too

**Comment by kkilobyte** [comment]
i got kde wayland to boot but even the cursor lags (for 30 seconds) and everything becomes 100x slower which makes wayland not even worth using

**Comment by kkilobyte** [comment]
NEVERMIND, 

The `i965` driver worked after being enabled and rebooted, and then logging into a ***Wayland** session with smooth animations and zero screen tearing*, although ***X11** still presents a screen tearing effect with sluggish or no animations*.

The `i915` driver is completely borked on both Wayland and X11 however i915 is meant for Apollolake anyways iirc, (Celeron N4000 is Geminilake).

**Comment by kkilobyte** [comment]
Just for documentation purposes, I have tested on Celeron N4020 and N4000 varients of octopus phaser360 (Lenovo 300e Chromebook Gen 2 (Intel)) with the same issue.

**Comment by funkymokey** [comment]
How do you have KDE running welll? My chromebook has the exact same board and specifics, but when booting into a KDE shim my chromebook becomes extremely slow and laggy.

**Comment by goober3853** [comment]
> How do you have KDE running welll? My chromebook has the exact same board and specifics, but when booting into a KDE shim my chromebook becomes extremely slow and laggy.

sudo apt install libglx-amber0 libegl-amber0 && echo "MESA_LOADER_DRIVER_OVERRIDE=i965" | sudo tee -a /etc/environment

Ripped from kkilobyte's bug report, I have an octopus board and this seemed to fix it.
